toolchain/lower/file_context.cpp

```diff
@@ -332,6 +561,24 @@ auto FileContext::BuildFunctionDecl(SemIR::FunctionId function_id,
   Mangler m(*this);
   std::string mangled_name = m.Mangle(function_id, specific_id);
 
+  // Create unique fingerprint for the function type
```

Comment:
Minor typo suggestion, but you might consider rewrapping with the full paragraph.

```suggestion
  // Create a unique fingerprint for the function type.
```